feature/experimental/bgp/otg_tests/bgp_tcp_mss_path_mtu/bgp_tcp_mss_path_mtu_test.go

```diff
@@ -0,0 +1,437 @@
+
+package bgp_tcp_mss_path_mtu_otg_test
+
+import (
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/open-traffic-generator/snappi/gosnappi"
+	"github.com/openconfig/featureprofiles/internal/attrs"
+	"github.com/openconfig/featureprofiles/internal/deviations"
+	"github.com/openconfig/featureprofiles/internal/fptest"
+	"github.com/openconfig/ondatra"
+	"github.com/openconfig/ondatra/gnmi"
+	"github.com/openconfig/ondatra/gnmi/oc"
+	"github.com/openconfig/ondatra/otg"
+	"github.com/openconfig/ygnmi/ygnmi"
+	"github.com/openconfig/ygot/ygot"
+)
+
+func TestMain(m *testing.M) {
+	fptest.RunTests(m)
+}
+
+// The testbed consists of ate:port1 -> dut:port1 and
+// dut:port2 -> ate:port2.  The first pair is called the "source"
+// pair, and the second the "destination" pair.
+//
+// 	ATE   Port-1 192.0.2.2 --------  DUT-1 Port-1 192.0.2.1
+//	DUT-1 Port-2 192.0.2.5 --------  DUT-2 Port-1 192.0.2.6
+
+const (
+	peerGrpName1    = "BGP-PEER-GROUP1"
+	dut1AS          = 65501
+	ateAS1          = 65502
+	dut2AS          = 65502
+	plenIPv4        = 30
+	plenIPv6        = 126
+	mtu5040B        = uint16(5040)
+	mtu4096B        = uint16(4096)
+	mtu512B         = uint16(512)
+	mtu1500B        = uint16(1500)
+	vlan10          = 10
+	vlan20          = 20
+	isisInstance    = "DEFAULT"
+	authPWd         = "BGPTCPMSS"
+	dut1AreaAddress = "49.0001"
+	dut1SysID       = "1920.0000.2001"
+	dut2AreaAddress = "49.0001"
+	dut2SysID       = "1920.0000.3001"
+	ateSysID        = "640000000001"
+)
+
+var (
+	dut1Port1 = attrs.Attributes{
+		Desc:    "DUT to ATE source",
+		IPv4:    "192.0.2.1",
+		IPv6:    "2001:db8::192:0:2:1",
+		IPv4Len: plenIPv4,
+		IPv6Len: plenIPv6,
+	}
+	atePort1 = attrs.Attributes{
+		Name:    "ateSrc",
+		IPv4:    "192.0.2.2",
+		IPv6:    "2001:db8::192:0:2:2",
+		MAC:     "02:00:01:01:01:01",
+		IPv4Len: plenIPv4,
+		IPv6Len: plenIPv6,
+	}
+	dut1Port2 = attrs.Attributes{
+		Desc:    "DUT1 to DUT2",
+		IPv4:    "192.0.2.5",
+		IPv4Len: plenIPv4,
+	}
+	dut2Port1 = attrs.Attributes{
+		Name:    "DUT2 to DUT1",
+		IPv4:    "192.0.2.6",
+		IPv4Len: plenIPv4,
+	}
+)
+
+// configureDUT configures all the interfaces on the DUT.
+func configureDUT(t *testing.T) {
+	dc := gnmi.OC()
+	dut1 := ondatra.DUT(t, "dut1")
+	dut2 := ondatra.DUT(t, "dut2")
+
+	t.Log("Configure interfaces on dut1.")
+	i1 := dut1Port1.NewOCInterface(dut1.Port(t, "port1").Name(), dut1)
+	gnmi.Replace(t, dut1, dc.Interface(i1.GetName()).Config(), i1)
+	i2 := dut1Port2.NewOCInterface(dut1.Port(t, "port2").Name(), dut1)
+	gnmi.Replace(t, dut1, dc.Interface(i2.GetName()).Config(), i2)
+
+	t.Log("Configure interfaces on dut2.")
+	i3 := dut2Port1.NewOCInterface(dut2.Port(t, "port1").Name(), dut2)
+	gnmi.Replace(t, dut2, dc.Interface(i3.GetName()).Config(), i3)
+}
+
+// bgpCreateNbr creates bgp configuration on dut device.
+func bgpCreateNbr(t *testing.T, dut *ondatra.DUTDevice, authPwd, routerId string, localAs uint32, nbrs []*bgpNeighbor) *oc.NetworkInstance_Protocol {
+
```

Comment:
nit: for style consistency, no blank line at the beginning of the method body here, and below

Reply:
updated.